### ISSUE TITLE
SendGrid sending plain text email as html

### ISSUE THREAD

**Issue by rtvenge** [awesomemotive/WP-Mail-SMTP#7]
While using the SendGrid Mailer, plain text emails (in my case, password reset emails) are being rendered as html emails, stripping out the reset link and the line breaks.

## Plugin version

1.2.5

## Current Behavior

`text/plain` emails are being sent as `text/html`.

## Expected Behavior
These emails should be sent as `text/plain`.

## Possible Solution

After looking at the `$this->set_content()` function, it looks like it expects a string for `text/plain` emails. To fix this, you could check the ContentType first:

```
if ($this->phpmailer->ContentType == 'text/html') {
  $this->set_content(
    array(
      'html' => $this->phpmailer->Body,
      'text' => $this->phpmailer->AltBody,
    )
  );
} else {
  $this->set_content($this->phpmailer->Body);
}
```

Not sure this is the _best_ solution, but it did fix the issue for me without restructuring the `$this->set_content()` functions on every Provider.

## Steps to Reproduce (for bugs)

1. Set Provider to SendGrid.
2. Log out of Wordpress site
3. Go to login screen and click "Lost your password"
4. Enter email address of a known user.
5. Receive email from Wordpress in stall with line breaks stripped and password reset link stripped (because it's in `<>`'s).

## Screenshots (if appropriate)

<img width="1218" alt="sendgrid-email" src="https://user-images.githubusercontent.com/303836/41487909-61688f2a-709f-11e8-8d44-ecac15b30abd.png">

**Comment by slaFFik** [comment]
Thank you for posting this!

This is a known issue and will be fixed in the upcoming WP Mail SMTP 1.3.

**Comment by slaFFik** [comment]
Fixed in #8.

Thank you for your contribution! This fix will be part of WP Mail SMTP 1.3.0.
